shell/platform/darwin/ios/framework/Source/FlutterTextInputPlugin.mm

```diff
@@ -548,6 +548,7 @@ - (void)hideTextInput {
 
 - (void)setTextInputClient:(int)client withConfiguration:(NSDictionary*)configuration {
   _view.keyboardType = ToUIKeyboardType(configuration[@"inputType"]);
+  _view.secureTextEntry = [@"true" isEqualToString:configuration[@"obscureText"]];
```

Comment:
Good point -- serialized that way in flutter/flutter#9908, but I forgot json has bools. Will send an update on the framework side and update this PR.